schema.graphql

```diff
@@ -177,6 +177,30 @@ type Recommendation @key(fields: "item { itemId }") {
     recSrc: String!
 }
 
+"""
+Represents a Recomendation from the curated corpus
+"""
+type CorpusRecommendation @key(fields: "id") @key(fields: "corpusItem { id }") @key(fields: "item { url }") {
+    """
+    A unique identifier for this specific recommendation across all recommendations. This identifier is generated by the
+    a machine learning system making recommendation decisions. This field can be joined with recommendation decisions to
+    aggregate more meta data about the decision.
+    """
+    id: ID!
+
+    """
+    URL of the corpus item.
+    """
+    url: String!
+
+    """
+    Corpus item id.
+    """
+    corpusItemId: String!
+}
+
+union Recommendation = ParserRecommendation | CorpusRecommendation
```

Comment:
Apparently you cannot change a type from a native type to a union type without possibly impacting clients.